cli/src/main/java/net/neoforged/jst/cli/SourceFileProcessor.java

```diff
@@ -63,16 +66,23 @@ public boolean process(FileSource source, FileSink sink, List<SourceTransformer>
                 });
             }
         } else {
+            AtomicBoolean status = new AtomicBoolean(true);
             try (var asyncOut = new OrderedParallelWorkQueue(sink, maxQueueDepth);
                  var stream = source.streamEntries()) {
                 stream.forEach(entry -> asyncOut.submitAsync(parallelSink -> {
                     try {
-                        processEntry(entry, sourceRoot, transformers, parallelSink);
+                        var isOk = processEntry(entry, sourceRoot, transformers, parallelSink);
+                        if (!isOk) {
+                            status.set(false);
+                        }
```

Comment:
Currently I have `@return {@code true} if the transformation should continue, {@code false} otherwise` -- I'm not sure how to clarify further the semantics. Returning false causes the transformation of that single class to fail. JST will fail if any issues occur while transforming files. Do you have a suggested way of phrasing that?

Reply:
Oh, wait, I think I got it -- is the confusion that "the transformation won't continue" doesn't necessarily imply that something will fail? I can fix that on this (and the other javadoc that uses similar wording) then